spec/tapioca/dsl/compilers/protobuf_spec.rb

```diff
@@ -253,6 +268,39 @@ def indices=(value); end
               assert_equal(expected, rbi_for(:Cart))
             end
 
+            it "generates methods in RBI files for classes with Protobuf with non-optional integer field type" do
+              add_proto_file("cart", <<~PROTO)
+                syntax = "proto3";
+
+                message Cart {
+                  int32 shop_id = 1;
+                }
+              PROTO
+
+              expected = <<~RBI
+                # typed: strong
+
+                class Cart
+                  sig { params(shop_id: T.nilable(Integer)).void }
+                  def initialize(shop_id: nil); end
+
+                  sig { void }
+                  def clear_shop_id; end
+
+                  sig { returns(T.any(Integer, T::Boolean)) }
+                  def has_shop_id?; end
```

Comment:
I thought we weren't supposed to generate `has_shop_id?` method since it is not optional. What am I missing about this expectation?

Reply:
Thanks, fixed per the comment above.